src/core/acroform/PDFAcroField.ts

```diff
@@ -96,6 +110,24 @@ class PDFAcroField {
     return DA?.asString();
   }
 
+  setFontSize(fontSize: number) {
+    if (this.DA()) {
+      const da = this.getDefaultAppearance()!;
+      const daMatch = findLastMatch(da, tfRegex);
```

Comment:
Can we do the following to make the null check safer and avoid the non-null assertion?
```suggestion
  setFontSize(fontSize: number) {
    const da = this.getDefaultAppearance();
    if (da) {
      const daMatch = findLastMatch(da, tfRegex);
```